backend/src/main/java/com/pokemon/pokedex/model/PokemonBasicResource.java

```diff
@@ -0,0 +1,13 @@
+package com.pokemon.pokedex.model;
+
+import java.util.List;
+import lombok.Value;
+
+@Value
```

Comment:
What's the difference using `@Value` or using `@Data`?

Reply:
@Value is inmutable (it doesn't include setters) so i try/prefer to use it before using @Data